public/scripts/extensions/vectors/settings.html

```diff
@@ -132,33 +132,33 @@
             <div class="flex-container">
                 <label class="checkbox_label expander" for="vectors_include_wi" title="Query results can activate World Info entries.">
                     <input id="vectors_include_wi" type="checkbox" class="checkbox">
-                    Include in World Info Scanning
+                    <div data-i18n="Include in World Info Scanning">Include in World Info Scanning</div>
                 </label>
             </div>
 
             <hr>
 
-            <h4>
+            <h4 data-i18n="World Info settings">
                 World Info settings
             </h4>
 
             <label class="checkbox_label" for="vectors_enabled_world_info" title="Enable activation of World Info entries based on vector similarity.">
                 <input id="vectors_enabled_world_info" type="checkbox" class="checkbox">
-                Enabled for World Info
+                <div data-i18n="Enable for World Info">Enable for World Info</div>
```

Comment:
Replace div with span

Reply:
fixed in https://github.com/SillyTavern/SillyTavern/pull/2376/commits/12c5466f08555ea88e350ba5aff0572878e44ba7